vitessce/config_converter.py

```diff
@@ -0,0 +1,362 @@
+        - The expression matrix
+        - The cell metadata
+        - The coordinates
+        The name of each of those files is read from the cellbrowser configuration.
+        """
+        self._load_expr_matrix()
+        self._load_cell_metadata()
+        self._load_coordinates()
+        self._filter_data()
+
+    def export_anndata_object(self):
+        """
+        Writes the contents of the Anndata object of the class instance to a Zarr store. Saves the Zarr store to the output directory
+        that the class is instantiated with.
+        """
+        data_dir = join(self.output_dir, self.project_name)
+        zarr_filepath = join(data_dir, "out.adata.zarr")
+
+        obs_columns_list = self.adata.obs.columns.tolist()
+        obsm_keys = list(self.adata.obsm.keys())
+        var_cols_list = list(self.adata.var.keys())
+        print("About to write the Anndata object to the Zarr store. The following properties will be saved:")
+        print(f"  Obs columns: {obs_columns_list}")
+        print(f"  Obsm keys: {obsm_keys}")
+        print(f"  Var columns: {var_cols_list}")
+
+        # This is done because of the optimize_adata function, which expects the obsm key to be numpy type
+        for key in self.adata.obsm.keys():
+            if isinstance(self.adata.obsm[key], pd.DataFrame):
+                print(f"obsm {key} is an instance of DataFrame, converting it to numpy array.")
+                self.adata.obsm[key] = self.adata.obsm[key].to_numpy()
+
+        self.adata = optimize_adata(
+            self.adata,
+            obs_cols=obs_columns_list,
+            obsm_keys=obsm_keys,
+            optimize_X=True,
+            var_cols=var_cols_list,
+        )
+
+        os.makedirs(os.path.dirname(data_dir), exist_ok=True)
+        self.adata.write_zarr(zarr_filepath, chunks=[self.adata.shape[0], VAR_CHUNK_SIZE])
+        print("Successfully saved Anndata object to the Zarr store.")
+
+    def create_vitessce_config(self):
+        """
+        Saves the Anndata object to a file and creates a Vitessce configuration dictionary for the project.
+        """
+        output_dir = join(self.output_dir, self.project_name)
+        os.makedirs(os.path.dirname(output_dir), exist_ok=True)
+
+        adata_path = join(self.output_dir, self.project_name, "out.adata.h5ad")
+        self.adata.write(adata_path)
+
+        path = Path(adata_path)
+        anndata_wrapper_inst = AnnDataWrapper(
+            adata_path=str(path.parent),
+            obs_embedding_names=list(self.adata.obsm.keys()),
+            obs_set_names=self.adata.obs.columns.tolist()
+        )
+        vc = VitessceConfig("1.0.15", f"Vitessce configuration for CellBrowser project {self.project_name}")
+        anndata_wrapper_inst.auto_view_config(vc)
+        vc.get_datasets()[0].add_file(
+            file_type="anndata.h5ad",
+            url=str(path)
+        )
```

Comment:
The AnnDataWrapper should take care of adding as a dataset - the URL also needs to be a URL rather than a Path for using `add_file` https://github.com/vitessce/vitessce-python/blob/f8d860411ef79dc95f3c89715edd92c67a13b9b9/vitessce/wrappers.py#L664
```suggestion
```

Reply:
The AnnDataWrapper adds a dataset, but it is configured with empty list as files. This is why I attempted to add a file manually, but now I think this should be removed, because I am not adding the correct URL in here anyway.

Reply:
If the AnnDataWrapper.auto_view_config function is not working as expected then we do not need to worry about it in this PR (can make a separate issue to fix it)

Reply:
Sure, I have created issue #261